src/inspector/parsers/cbor/parser.ts

```diff
@@ -16,8 +17,8 @@ export function safeCBORParse(credential: string): Result<ParsedCBOR> {
     return {
       kind: 'error',
       error: {
-        name: 'No byteformat detected',
-        message: 'Not one of the supported byte formats (hex, base64url, base64)',
+        name: 'CBOR Parse Error',
+        message: 'No one of the supported byte formats (hex, base64url, base64) given',
```

Comment:
```suggestion
        message: 'None of the supported byte formats (hex, base64url, base64) given',
```
Could we change it to this?